recipes/msms/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "2.6.1" %}
+{% set url_base = "http://mgltools.scripps.edu/downloads/tars/releases/MSMSRELEASE" %}
+
+package:
+  name: msms
+  version: {{ version }}
+
+source:
+  - url: {{ url_base }}/REL{{ version }}/msms_i86_64Linux2_{{ version }}.tar.gz  # [linux]
+    md5: 650d8c6fd49d9ce291f30ed530d5f313  # [linux]
+  - url: {{ url_base }}/REL{{ version }}/msms_MacOSX_{{ version }}.tar.gz  # [osx]
+    md5: 36bd8447908d12a8703f458564cbe74b  # [osx]
+  - url: {{ url_base }}/REL{{ version }}/msms_win32_{{ version }}.zip  # [win]
+    md5: 482c38ca00a2c091cdc0807a8a272386  # [win32]
```

Comment:
Please build from source

Reply:
What does that mean exactly? Scrips has only released this code as precompiled executables. I have modeled my recipe after the one in bioconda: 
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/msms/meta.yaml

Reply:
probably worth asking them

Reply:
I have reached out to the authors and maintainers at Scripps (through MGL Tools) several times over the last year, but I have yet to hear back. I have also checked distribution for tools that rely on this executable. CCP4, VMD, Biopython, and PyMOL all can check for the presence of MSMS and read its output but do not distribute the executable itself. I would be comfortable using the Bioconda channel's version except that it excludes Windows compatibility.

What steps can I take to move this forward?

Reply:
Yeah this seems to be the only way then.  Can you please use sha256 for your hashes here, though.  I think the various bots might break if you use md5